lib/animina/accounts/resources/user.ex

```diff
@@ -201,7 +191,9 @@ defmodule Animina.Accounts.User do
         :confirmed_at,
         :is_in_waitlist,
         :registration_completed_at,
-        :country
+        :country,
+        :confirmation_pin,
```

Comment:
I don't like the idea of having the `confirmation_pin` stored in clear in the database. Please store a secure hash sum and compare that with the entered PIN.

Reply:
Okay ,  kindly check [ Bcypt Documentation]( https://hexdocs.pm/bcrypt_elixir/Bcrypt.html) and  the comment below and advise , We need a way to store it to be able to send it to the user if they request for additional emails